packages/treasury/test/test-interest.js

```diff
@@ -28,249 +33,283 @@ test('too soon', async t => {
 
 test('basic charge 1 period', async t => {
   const { brand } = makeIssuerKit('ducats');
+  const annualRate = makeRatio(250, brand, 10000n);
   const calculator = makeInterestCalculator(
     brand,
-    makeRatio((1n * SECONDS_PER_YEAR) / 3n, brand),
-    3n,
-    6n,
+    annualRate,
+    ONE_DAY,
+    ONE_MONTH,
   );
   const debtStatus = {
     newDebt: amountMath.make(100000n, brand),
-    latestInterestUpdate: 10n,
+    latestInterestUpdate: 0n,
+    interest: amountMath.make(0n, brand),
   };
-  t.deepEqual(calculator.calculate(debtStatus, 13n), {
-    latestInterestUpdate: 13n,
-    interest: amountMath.make(1000n, brand),
-    newDebt: amountMath.make(101000n, brand),
+  t.deepEqual(calculator.calculate(debtStatus, ONE_DAY), {
+    latestInterestUpdate: ONE_DAY,
+    interest: amountMath.make(6n, brand),
+    newDebt: amountMath.make(100006n, brand),
   });
 });
 
 test('basic 2 charge periods', async t => {
   const { brand } = makeIssuerKit('ducats');
+  const annualRate = makeRatio(250, brand, 10000n);
   const calculator = makeInterestCalculator(
     brand,
-    makeRatio((1n * SECONDS_PER_YEAR) / 3n, brand),
-    3n,
-    6n,
+    annualRate,
+    ONE_DAY,
+    ONE_MONTH,
   );
   const debtStatus = {
```

Comment:
comment: test interest between the 1st day and the 3rd day, which should be 6 (daily interest) * 2